.travis.yml

```diff
@@ -178,6 +178,16 @@ jobs:
       after_script:
         - ./scripts/ci/job_hooks/after_e2e.sh
 
+    - stage: e2e
+      name: 'Test Suites: Folder Rules - Playwright'
+      before_script:
+        - ./scripts/ci/job_hooks/before_e2e.sh "$S3_DBP_FOLDER/alfresco-content-app.tar.bz2" "$CONTENT_CE_DIST_PATH" "-a" || travis_terminate 1
+      script:
+        - npm install -D @playwright/test
```

Comment:
don't do this in the travis, it should be coming from the package.json

Reply:
Ran out of options so decided to debug the issue this way :)